integration

```diff
@@ -640,6 +640,7 @@
   "lindell/home-assistant-svt-play",
   "lindell/home-assistant-tv4-play",
   "linsvensson/sensor.greenely",
+  "litinoveweedle/SmartIR",
```

Comment:
<https://github.com/litinoveweedle/SmartIR/actions/runs/12643153531/job/35228688394#step:4:4>
misrepresented in the description
<img width="614" alt="2025-02-27_08-20-09" src="https://github.com/user-attachments/assets/cd7989f4-65e2-4258-aaba-6d7ec3d49f68" />